build.sbt

```diff
@@ -0,0 +1,10 @@
+import Settings._
+
+lazy val root = project.in(file("automation"))
+  .settings(rootSettings:_*)
+
+name := "root"
+
+version := "1.0"
+
+scalaVersion := "2.11.8"
```

Comment:
Yes, unfortunately updating scala, scalatest, and project/dependency clean-up will have to come later. I definitely want to do that soon, though.